source/languages/en/riak/dev/taste-of-riak/querying.md

```diff
@@ -44,10 +44,11 @@ Please select the language with which you'd like to proceed:
 
 <ul class="planguages">
 <li><a href="/dev/taste-of-riak/querying-java/"><img src="/images/plangs/java.jpg" alt="Java"></a></li>
-<li><a href="/dev/taste-of-riak/querying-erlang/"><img src="/images/plangs/erlang.jpg" alt="Erlang"></a></li>
 <li><a href="/dev/taste-of-riak/querying-ruby/"><img src="/images/plangs/ruby.jpg" alt="Ruby"></a></li>
-<li><a href="/dev/taste-of-riak/querying-php/"><img src="/images/plangs/php.png" alt="PHP"></a></li>
 <li><a href="/dev/taste-of-riak/querying-python/"><img src="/images/plangs/python.png" alt="Python"></a></li>
 <li><a href="/dev/taste-of-riak/querying-csharp/"><img src="/images/plangs/csharp.png" alt="CSharp"></a></li>
+<li><a href="/dev/taste-of-riak/querying-nodejs/"><img src="/images/plangs/nodejs.png" alt="NodeJS"></a></li>
+<li><a href="/dev/taste-of-riak/querying-erlang/"><img src="/images/plangs/erlang.jpg" alt="Erlang"></a></li>
+<li><a href="/dev/taste-of-riak/querying-php/"><img src="/images/plangs/php.png" alt="PHP"></a></li>
```

Comment:
Yes I am trying to be consistent in ordering at all times:

* Java
* Ruby
* Python
* C#
* Node
* Erlang
* PHP